File: src/main/java/com/mrpowergamerbr/loritta/commands/nashorn/NashornUtils.java
package com.mrpowergamerbr.loritta.commands.nashorn;

import com.github.kevinsawicki.http.HttpRequest;
import com.mrpowergamerbr.loritta.nashorn.wrappers.NashornImage;
import com.mrpowergamerbr.loritta.utils.Constants;
import com.mrpowergamerbr.loritta.utils.LorittaUtils;

import java.awt.*;
import java.awt.image.BufferedImage;
import java.io.IOException;

/**
 * Classe de utilidades para comandos usando o Nashorn
 */
public final class NashornUtils {
	private NashornUtils() {
	}

	public static String loritta() { // Método teste
		return "Loritta!";
	}

	public static String getURL(String url) {
		return HttpRequest.get(url).userAgent(Constants.USER_AGENT).body();
	}

	public static NashornImage downloadImage(String url) throws IOException {
		BufferedImage image = LorittaUtils.downloadImage(url);
		NashornImage nashornImage = new NashornImage(image);
		return nashornImage;
	}

	public static Color createColor(int r, int g, int b) throws IOException {
		return new Color(r, g, b);
	}
}

File: src/main/java/com/mrpowergamerbr/loritta/commands/nashorn/NashornUtils.kt
package com.mrpowergamerbr.loritta.commands.nashorn

import java.awt.Color
import java.io.IOException

/**
 * Classe de utilidades para comandos usando o Nashorn
 */
object NashornUtils {
	@JvmStatic
	fun loritta(): String { // Método teste
		return "Loritta!"
	}

	@Throws(IOException::class)
	@JvmStatic
	fun createColor(r: Int, g: Int, b: Int): Color {
		return Color(r, g, b)
	}
}